kubernetes/config/prometheus/monitor.yaml

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Is this automatically created or did you try the Prometheus integration?

Reply:
This is from skeleton for future extension.

Reply:
Thanks, I suggest leaving it out to keep this PR shorter (if that's not too much work for you)

Reply:
Got it. I will delete ones not being used.